documentation/modules/managing/con-producer-config-properties.adoc

```diff
@@ -55,25 +56,59 @@ acks=all <1>
 ----
 
 <1> `acks=all` forces a partition leader to replicate messages to a certain number of followers before
-acknowledging that the message request was successfully received.
+acknowledging that the message request was successfully received. 
 
 The `acks=all` setting offers the strongest guarantee of delivery, but it will increase the latency between the producer sending a message and receiving acknowledgment.
 If you don't require such strong guarantees, a setting of `acks=0` or `acks=1` provides either no delivery guarantees or only acknowledgment that the leader partition has written the record to its log.
 
-The number of brokers which need to have appended the messages to their logs before the acknowledgment is sent to the producer is determined by the topic's `min.insync.replicas` configuration.
-A typical starting point is to have a topic replication factor of 3, with two in-sync replicas on other brokers.
-In this configuration, the producer can continue unaffected if a single broker is unavailable.
-If a second broker becomes unavailable, the producer won’t receive acknowledgments and won’t be able to produce more messages.
+With `acks=all`, a leader waits for all in-sync replicas to acknowledge message delivery.
+A topic's `min.insync.replicas` configuration sets the minimum required number of in-sync replica acknowledgements.
+
+A typical starting point is to use the following configuration:
+
+* Producer
+** `acks=all`
+* Topic:
+** `default.replication.factor=3`
+** `min.insync.replicas=2`
 
 .Topic configuration to support `acks=all`
 [source,env]
 ----
 # ...
+default.replication.factor=3
 min.insync.replicas=2 <1>
 # ...
 ----
 <1> Use `2` in-sync replicas. The default is `1`.
 
+A topic replication factor of 3 creates one leader partition and two followers.
+In this configuration, the producer can continue if a single replica broker is unavailable.
+Some delay can occur whilst a failed broker is removed from the in-sync replicas or a new leader is created.
+If a second replica broker becomes unavailable, the producer won’t receive acknowledgments and won’t be able to produce more messages.
+
+.Replica broker availability
+[cols="2,3,2",options="header"]
+|===
+
+|Number of replica brokers available
```

Comment:
I think you mean available and in-sync for this partition

Reply:
I've moved around to make it more obvious the example config is related to the table